src/Reveal.Sdk.Dom/Visualizations/AreaChartVisualization.cs

```diff
@@ -3,12 +3,24 @@
 
 namespace Reveal.Sdk.Dom.Visualizations
 {
+    /// <summary>
+    /// The area chart visualization renders as a collection of points connected by straight line segments with the area below the line filled in. Values are represented on the y-axis (labels on the left side) and categories are displayed on the x-axis (bottom labels). This chart emphasize the amount of change over a period of time or compare multiple items as well as the relationship of parts of a whole by displaying the total of the plotted values. Therefore, they are often chronological, showing a change of quantity e.g. accumulation of a commodity over time.

```

Comment:
remove anything within the parenthesis. At this point people know what an x and y axis are and where they exist on a chart.